utility/test/actor_test.go

```diff
@@ -344,59 +353,32 @@ func TestUtilityContext_GetMessageEditStakeSignerCandidates(t *testing.T) {
 			require.Equal(t, len(candidates), 2, "unexpected number of candidates")
 			require.Equal(t, actor.GetOutput(), hex.EncodeToString(candidates[0]), "incorrect output candidate")
 			require.Equal(t, actor.GetAddress(), hex.EncodeToString(candidates[1]), "incorrect addr candidate")
+
 			test_artifacts.CleanupTest(ctx)
 		})
 	}
 }
 
-func TestUtilityContext_UnstakesPausedBefore(t *testing.T) {
```

Comment:
`TestUtilityContext_UnstakePausedBefore` covers this logic for all actor while `TestUtilityContext_UnstakesPausedBefore` only covers is for applications, and I believe is legacy code.

TestUtilityContext_UnstakesPausedBefore had a few issues:
1. Lack of casting on a line such as `require.Equal(t, -1, actor.GetUnstakingHeight())` (357)
2. The check on line 364 (`require.Equal(t, -1, actor.GetUnstakingHeight())`) was incorrect and simply had to be removed because on line 367 we do the right thing.

I fixed the tests initially to make it pass, but later realized that it's "legacy" relative to `TestUtilityContext_UnstakePausedBefore`.

Lmk if that makes sense.